apis/config/v1alpha1/featuregate_webhook_test.go

```diff
@@ -337,9 +337,9 @@ func TestNamespaceConflicts(t *testing.T) {
 // This function is intended to be used in tests for comparing expected and actual values, and printing the diff for
 // users to debug:
 //
-//   if diff := sliceDiffIgnoreOrder(got, want); diff != "" {
-//       t.Errorf("got: %v, want: %v, diff: %s", got, want, diff)
-//   }
+//	if diff := sliceDiffIgnoreOrder(got, want); diff != "" {
```

Comment:
When will this get un-commented?

Reply:
This is just a godoc comment showing an example. Actual function isn't commented out.